scripts/ubuntu/install_docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Uninstall old versions
+sudo apt-get remove docker docker-engine docker.io containerd runc
+
+# Set up the repository
+sudo apt-get update
+sudo apt-get install \
```

Comment:
Is it better to add `-y`?

Reply:
I think it would be better to tell the user when their system going to be updated even by apt. So I prefer not using -y option.